src/translib/intf_app.go

```diff
@@ -438,7 +458,7 @@ func (app *IntfApp) doGetAllIpKeys(d *db.DB, dbSpec *db.TableSpec) ([]db.Key, er
 func (app *IntfApp) getSpecificAttr(targetUriPath string, ifKey string, oc_val *ocbinds.OpenconfigInterfaces_Interfaces_Interface_State) (bool, error) {
 	switch targetUriPath {
 	case "/openconfig-interfaces:interfaces/interface/state/oper-status":
-		val, e := app.getIntfAttr(ifKey, PORT_OPER_STATUS)
+		val, e := app.getIntfAttr(ifKey, PORT_OPER_STATUS, "ifTableMap")
```

Comment:
Would it be better to have mapType enum and pass the enum, rather than passing a string?

Reply:
Addressed